### PR TITLE
Remove unused config values and folder binds

### PR DESCRIPTION
**Pull request by talavis**
* Remove unused config variables:
  - `LOCAL_TEMP_CACHE`
  - `DOWNLOAD_FOLDER`
  - `UPLOAD_FOLDER`
* Remove corresponding folder binds in `docker-compose.yml`
* Change the value of `MAX_CONTENT_LENGTH` to hex to make it clearer
* Remove reference to `.dds-cli.json` in `docker-compose.yml`

### CODE REVIEW

File: dds_web/config.py [hunk 1]
```diff
@@ -25,15 +25,12 @@ class Config(object):
...
+    MAX_CONTENT_LENGTH = 0x1000000
     MAX_DOWNLOAD_LIMIT = 1000000000
 
     # Expected paths - these are the bind paths *inside* the container
     USE_LOCAL_DB = True
     LOGS_DIR = "/dds_web/logs"
-    UPLOAD_FOLDER = "/dds_web/uploads"
-    DOWNLOAD_FOLDER = "/dds_web/downloads"
-    LOCAL_TEMP_CACHE = "/dds_web/local_temp_cache"
     DDS_S3_CONFIG = "/code/dds_web/sensitive/s3_config.json"
```
Comment (alneberg) [inline dds_web/config.py:34]: @inaod568 we can remove the `DDS_S3_CONFIG` as well right?

**Review by alneberg — APPROVED** [review]
Looks correct to me!

File: dds_web/config.py (hunk 1 above)
Comment (talavis) [reply dds_web/config.py:34]: That should mean that I can remove `s3_config.json? as well from the comment in `docker_compose.yml`?

Comment (alneberg) [reply dds_web/config.py:34]: Yes it would. I think we simply moved the values from that one into the general config file.

Comment (inaod568) [reply dds_web/config.py:34]: Yes 👍🏻

**Review by inaod568 — APPROVED** [review]

**Pull request merged**